yarn.lock

```diff
@@ -1253,7 +1253,7 @@
   resolved "https://registry.yarnpkg.com/@emotion/hash/-/hash-0.8.0.tgz#bbbff68978fefdbe68ccb533bc8cbe1d1afb5413"
```

Comment:
Try not to change dependencies if not related to the PR—adds noise to reviews and tests failures.

Reply:
I think this points to a bigger issue across our repos -- each of us developers seem to use different versions of node and each of our repos run on different versions of node.  If we want to minimize lock changes I think we should implement AVN and lock the node version in each of our repos to the version of node we build with.